.github/workflows/test.yml

```
@@ -44,7 +44,8 @@ jobs:
       run: cargo --version
     - name: Test
       run:
```

```suggestion
      run: |
```
Without the `|` the command seems to be interpreted as `cargo test --lib -- --nocapture cargo test --doc -- --nocapture`.

---

Uh yeah, sorry. Already fixed that but didn't commit it yet. The github integration is not complete yet.